spring-cloud-dataflow-docs/src/main/asciidoc/howto.adoc

```diff
@@ -197,4 +201,124 @@ dataflow:>stream deploy foo --properties "deployer.transform.count=2,app.transfo
+dataflow:>stream create --name mydata --definition "time | jdbc --spring.datasource.url=jdbc:mysql://localhost:3306/test --spring.datasource.username=root --spring.datasource.password=root --spring.datasource.driver-class-name=org.mariadb.jdbc.Driver" --deploy
+----
+
+For this to work, you'd have to have the following table in the MySQL database.
+
+[source,sql]
+----
+CREATE TABLE test.messages
+(
+  payload varchar(255)
+);
+----
+
+[source,bash]
+----
+mysql> desc test.messages;
++---------+--------------+------+-----+---------+-------+
+| Field   | Type         | Null | Key | Default | Extra |
++---------+--------------+------+-----+---------+-------+
+| payload | varchar(255) | YES  |     | NULL    |       |
++---------+--------------+------+-----+---------+-------+
+1 row in set (0.00 sec)
+----
+
+[source,bash]
+----
+mysql> select * from test.messages;
++-------------------+
+| payload           |
++-------------------+
+| 04/25/17 09:10:04 |
+| 04/25/17 09:10:06 |
+| 04/25/17 09:10:07 |
+| 04/25/17 09:10:08 |
+| 04/25/17 09:10:09 |
+.............
+.............
+.............
+----
+
+[[dataflow-multiple-brokers]]
+=== How to use multiple message-binders?
+For situations where the data is consumed and processed between two different message brokers, Spring
+Cloud Data Flow provides easy to override global configurations, out-of-the-box `bridge-processor`,
+and DSL primitives to build this type of topologies.
+
+Let's assume we have data queueing up in RabbitMQ _(e.g., queue = `fooRabbit`)_ and the requirement
+is to consume all the payloads and publish them to Apache Kafka _(e.g., topic = `barKafka`)_, as the
+destination for downstream consumption.
+
+Follow the global application of <<streams.adoc#spring-cloud-dataflow-global-properties, configurations>>
+to define multiple binder configurations.
+
+[source,properties]
+----
+# Apache Kafka Global Configurations (i.e., identified by "kafka1")
+spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.kafka1.type=kafka
+spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.kafka1.environment.spring.cloud.stream.kafka.binder.brokers=localhost:9092
+spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.kafka1.environment.spring.cloud.stream.kafka.binder.zkNodes=localhost:2181
+
+# RabbitMQ Global Configurations (i.e., identified by "rabbit1")
+spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.rabbit1.type=rabbit
+spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.rabbit1.environment.spring.rabbitmq.host=localhost
+spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.binders.rabbit1.environment.spring.rabbitmq.port=5672
+----
+
+NOTE: In this example, both the message brokers are running locally and reachable at `localhost`
+with respective ports.
+
+These properties can be supplied in a `.properties` file either to the server directly or via 
```

Comment:
reword `file either to the server directly` to `file that is accessible to the server directly or via`.